static/bidder-info/apacdex.yaml

```diff
@@ -0,0 +1,20 @@
+maintainer:
+  email: "ken@apacdex.com"
```

Comment:
This must be a group email address. Please review our [adapter dev docs](https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html).